README.md

```diff
@@ -17,39 +17,32 @@ use atoum;
 
 class Example extends atoum
 {
-    protected $blackfire;
+    private $blackfireClient;
 
     public function testExemple()
     {
-        $config = new \Blackfire\Profile\Configuration();
-        $config
-            ->assert('main.wall_time < 2s', "Temps d'execution")
-        ;
-
-        $callback = function() {
-
-            sleep(4);
-
-            //some code and/or atoum assertions
-            $this->boolean(true)->isTrue();
-        };
-
         $this
-            ->blackfireProfile($this->getBlackfireClient(), $callback, $config)
-            ->matchesAssertions()
+            ->blackfire($this->getBlackfireClient())
+                ->assert('main.wall_time < 2s', "Temps d'execution")
+                ->profile(function() {
+                    sleep(4);
+                    //some code of atoum assertions
+                    $this->boolean(true)->isTrue();
+                })
```

Comment:
It is indeed nicer, but only proxies the `assert()` method from `Configuration`. Defining custom metrics for instance is very common in such unit tests. In this case, you would need to fallback to using a proper configuration object. What about proxying all methods from `Configuration`? Or just add a `__call()` method to automatically proxy them?